### ISSUE TITLE
setup.py --hpy-abi=universal build_ext --inplace works only the first time

### ISSUE THREAD

**Issue by antocuni** [hpyproject/hpy#190]
This is a follow up of #140 . It seems that now `build_ext --inplace` works only the first time, then it fails because the stub file already exists:
```
$ cd proof-of-concept/
$ python setup.py --hpy-abi=universal build_ext -i   # first run
[...]

$ python setup.py --hpy-abi=universal build_ext -i   # second run 
[...]
error: ./pof.py already exists! Please delete.
```

/cc @hodgestar

**Comment by hodgestar** [comment]
I think we can just relax this assert and allow the stub file to be overwritten -- it is what happens with the .so file anyway.

**Comment by antocuni** [comment]
> I think we can just relax this assert and allow the stub file to be overwritten -- it is what happens with the .so file anyway.

yes, I agree. If we really want to be advanced, we could check whether the existing file is the same as the one we are going to write, and complain only if it's different.

**Comment by cklein** [comment]
What's about adding a comment similar to the comments added to the autogenerated C code?

```
# DO NOT EDIT THIS FILE!
# This file is automatically generated by hpy
#
# Run this to regenerate:
#     python setup.py --hpy-abi=universal build_ext -i```

**Comment by antocuni** [comment]
I think that adding a comment is a good is a good idea. I would not include the `run this to regenerate` part though, because it depends on how you build/install the package (e.g. it could be `setup.py build` or `pip install .`. And if the file ends up in a wheel, you can't regenerate it at all).

**Comment by hodgestar** [comment]
Fixed in #199.
